Aspen-Discovery/aspen-discovery
# feat: ILS usage graphs

Description

An Aspen admin can now view a usage graph and a raw data table for each of the variables included in the ILS Integration dashboard. This feature used to be limited to the System Reports dashboard, and it is now being extended.

Note: the implementation is based on the the System Reports existing usage graphs, and intends to mirror it where possible

Features:

The dashboard now includes links next to the title of each of its sections
Those links lead to the relevant Usage Graph page
The Usage Graph page contains both a graph and a raw data table
While the user is on a given ILS Integration Usage Graph page, the ILS Integration section on the left hand-side options menu remains open (active)

Test plan:

- apply the patch
- log in as an admin
- navigate to Administration Settings > ILS Integration > Dashboard
- click on the link displayed next to a section's header (title) and access the relevant Usage Graph page
- notice that the data displayed matches the summary found on the dashboard
